README.md

```diff
@@ -90,5 +90,26 @@ See [config defined in vscode plugin](https://github.com/nwolverson/vscode-ide-p
 
 ## Development
 
-See [vscode plugin](https://github.com/nwolverson/vscode-ide-purescript) repo. Common code via
-[purescript-ide-purescript-core](https://github.com/nwolverson/purescript-ide-purescript-core).
\ No newline at end of file
+To develop (rather than use) this language server
+
+1. Clone this repo and `npm install`
```

Comment:
You should mention psc-package install, or add it to the install script for the project.

Reply:
Thanks, added it to the scripts.